src/main/java/games/strategy/engine/config/PropertyReader.java

```diff
@@ -1,5 +1,4 @@
 package games.strategy.engine.config;
 
 public interface PropertyReader {
```

Comment:
Is there any reason to keep this interface around?  The only reference to it is from `PropertyFileReader` as an implemented interface.  Since it no longer has any methods, is that still necessary?  It doesn't appear to be used as a marker interface for anything.